hadoop-hdds/RocksDBCheckpointDiffer/README.md

```diff
@@ -0,0 +1,2 @@
+# RocksDiff
```

Comment:
TODO: Rename `RocksDBCheckpointDiffer` to `rocksdb-checkpoint-differ`

Reply:
done